docs/Home.md

```diff
@@ -45,4 +45,15 @@ Keep in mind, the features are based on _our team's_ requirements. In case the c
 
 Please note that the above examples are not definitive as new and existing libraries keep on emerging and evolving.
 
+Before an issue can be worked on, it must go through our voting process. 
+
+How do we vote on an issue? 🗳
+
+- If we agree with the RFC, we must react with 👍. 
+  - If there are 3 x 👍, then the issue is approved.
+- If we disagree with the RFC, we must react with 👎 and leave a comment explaining why. 
+  - If there are 3 x 👎, then the issue is rejected.
+- It is the responsibility of the RFC creator to label their proposed change as **essential** or **optional**.
```

Comment:
Should we move this to the top of the bullet point list?

Because the RFC creator will label their issue first, only after can the voting start 🙏